cluster/gce/gci/configure-helper.sh

```diff
@@ -1027,7 +1046,12 @@ function start-kube-proxy {
   fi
   local container_env=""
   if [[ -n "${ENABLE_CACHE_MUTATION_DETECTOR:-}" ]]; then
-    container_env="env:\n    - name: KUBE_CACHE_MUTATION_DETECTOR\n    value: \"${ENABLE_CACHE_MUTATION_DETECTOR}\""
+    # Daemonset manifest needs a different indent.
```

Comment:
The reason to be DaemonSet and Pod yaml files need different indents. Will make the indent a variable as suggested in https://github.com/kubernetes/kubernetes/pull/51227#discussion_r135127130. The real configuration hasn't changed.